### PR TITLE
AE_SHO-1419 - TX values for ESP-IDF v4.0 and v4.3

### PR DESCRIPTION
**Pull request by leo-ublox**

### CODE REVIEW

File: NINA-W1-OTP/components/ubx_phy_cal/ubx_phy_cal.c
Comment (jakkra) [inline NINA-W1-OTP/components/ubx_phy_cal/ubx_phy_cal.c]: Should only be one # here?

**Review by magnushanssonublox — APPROVED** [review]

**Comment by leo-ublox** [comment]
Draft parameters for IDF v4.3, waiting for confirmation

**Comment by jakkra** [comment]
Why did you remove the LIMIT stuff? If you remove that you should also remove https://github.com/u-blox/u-blox-sho-OpenCPU/blob/dccd7592592b927ab9d423193075c2a98fee4929/NINA-W1-OTP/components/ubx_phy_cal/ubx_phy_cal.c#L43 is as now it has no effect anymore.

File: NINA-W1-OTP/components/ubx_phy_cal/ubx_phy_cal.c
Comment (jakkra) [inline NINA-W1-OTP/components/ubx_phy_cal/ubx_phy_cal.c]: Still you have two ## here, should only be one

Comment (leo-ublox) [reply NINA-W1-OTP/components/ubx_phy_cal/ubx_phy_cal.c]: [f160ba7](https://github.com/u-blox/u-blox-sho-OpenCPU/pull/7/commits/f160ba7da989ed5aecd0df2c719202732f8c3cec)

**Review by jakkra — DISMISSED** [review]

**Comment by jakkra** [comment]
Just cleanup your commits before merge

**Review by jakkra — APPROVED** [review]

**Pull request merged**
